openapi.yaml

```diff
@@ -18416,6 +18453,63 @@ components:
                   type: string
                   x-stoplight:
                     id: njn2ukyaf5ra1
+    ContainerRegistryAssociatedServicesResponseList:
+      title: ContainerRegistryAssociatedServicesResponseList
+      x-stoplight:
+        id: xn1fsp17ahwfi
+      type: object
+      properties:
+        results:
+          $ref: '#/components/schemas/ContainerRegistryAssociatedServicesResponse'
+    ContainerRegistryAssociatedServicesResponse:
+      title: ContainerRegistryAssociatedServicesResponse
+      x-stoplight:
+        id: uxbkfqdj9zyi6
+      type: object
+      required:
+        - project_id
+        - project_name
+        - environment_id
+        - environment_name
+        - service_id
+        - service_name
+        - service_type
+      properties:
+        project_id:
+          type: string
+          x-stoplight:
+            id: 9jdx3t62nttjz
+          format: uuid
+        project_name:
+          type: string
+          x-stoplight:
+            id: cjlfx1ucomdv9
+        environment_id:
+          type: string
+          x-stoplight:
+            id: gk5r4dcoliuac
+          format: uuid
+        environment_name:
+          type: string
+          x-stoplight:
+            id: kxry5d624susz
+        service_id:
+          type: string
+          x-stoplight:
+            id: kmouxjf3vbtjo
+          format: uuid
+        service_name:
+          type: string
+          x-stoplight:
+            id: ga0yqhtmi4nl6
+        service_type:
+          $ref: '#/components/schemas/ContainerRegistryAssociatedServiceType'
+    ContainerRegistryAssociatedServiceType:
+      title: ContainerRegistryAssociatedServiceType
+      x-stoplight:
+        id: dif8pj1ys444y
+      enum:
+        - CONTAINER
```

Comment:
Why only a container ? Jobs can also be linked iiuc ?

Reply:
ah yes! I've forgotten it in the Core